Automattic/wp-calypso
# Revert "Components: <Animate>"

Reverts Automattic/wp-calypso#7713

Reverting since we're seeing the masterbar pop into place on non-chrome browsers

## Testing Instructions
- Load Calypso in FF, Chrome, Safari
- Does the masterbar no longer "pop" into place on refresh?

Test live: https://calypso.live/?branch=revert-7713-add/animate-component
